project-keeper/src/test/java/com/exasol/projectkeeper/validators/pom/PomFileValidatorTest.java

```diff
@@ -102,6 +109,21 @@ void testMissingGroupId() throws IOException {
         return validator.validate();
     }
 
+    @Test
+    void testFixParentVersion() throws IOException, XmlPullParserException {
+        final Parent parentWithWrongVersion = new Parent();
+        parentWithWrongVersion.setArtifactId("my-test-project-generated-parent");
+        parentWithWrongVersion.setGroupId(TestMavenModel.PROJECT_GROUP_ID);
+        parentWithWrongVersion.setVersion("3.2.0");
+        parentWithWrongVersion.setRelativePath("./pk_generated_parent.pom");
+        final TestMavenModel model = new TestMavenModel();
+        model.setParent(parentWithWrongVersion);
+        model.writeAsPomToProject(this.tempDir);
+        runFix(null);
```

Comment:
Is  `runFix(null)` a method of the fixture? Looks odd seems to have a side effect on the test but neither gets input nor output refs.

Reply:
yes, it runs on the tempDir. The tempdir is a temporary directory and the ref is stored as local field. So runFix can access it wiothout a ref.